common/src/main/java/com/wynntils/wynn/model/CharacterManager.java

```diff
@@ -53,6 +55,12 @@ public static boolean hasCharacter() {
     }
 
     public static CharacterInfo getCharacterInfo() {
+        if (currentCharacter == null) {
+            McUtils.sendMessageToClient(new TextComponent(
```

Comment:
This could spam the user imo this should be send only once at world join.

Reply:
Alright yea. Wynncraft itself sends a "you are using autojoin" message, so ill just look for that and then send the wynntils one i guess

Reply:
> This could spam the user imo this should be send only once at world join.

but in what way would it spam the user? Wouldnt it only send it to the user the first time he uses autojoin? The other times he would have to use the /class menu, and that would create the normal CharacterInfo

Reply:
Nevermind. The message is also sent the second the player does /class